exercises/practice/allergies/.meta/config.json

```diff
@@ -0,0 +1,19 @@
+{
+    "blurb": "Given a person's allergy score, determine whether or not they're allergic to a given item, and their full list of allergies.",
+    "source": "Jumpstart Lab Warm-up",
+    "source_url": "http://jumpstartlab.com",
+    "authors": [
+        "pclausen"
+    ],
+    "files": {
+        "solution": [
+            "allergies.f90"
+        ],
+        "test": [
+            "allergies_test.f90"
+        ],
+        "example": [
+            ".meta/example.f90"
+        ]
+    }
+}
```

Comment:
What would be line 19 is not a line, as it does not have the "EOL" at the end of it.

Reply:
I do not know how that has happend... Fixed